webhooks/src/main/java/org/openmbee/sdvc/webhooks/components/EventListener.java

```diff
@@ -0,0 +1,60 @@
+package org.openmbee.sdvc.webhooks.components;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.openmbee.sdvc.core.dao.WebhookDAO;
+import org.openmbee.sdvc.data.domains.global.Webhook;
+import org.openmbee.sdvc.core.objects.EventObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationListener;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+
+@Component
+public class EventListener implements ApplicationListener<EventObject> {
+
+    protected final Logger logger = LogManager.getLogger(getClass());
+
+    private WebhookDAO eventRepository;
+
+    @Autowired
+    public void setEventRepository(WebhookDAO eventRepository) {
+        this.eventRepository = eventRepository;
+    }
+
+    @Override
+    public void onApplicationEvent(EventObject eventObject) {
```

Comment:
for now to support system level webhook, add a env var that can specify a hook and post all events to it if it's configured